Dockerfile

```diff
@@ -197,7 +200,7 @@ RUN mkdir /var/run/fetchmail && chown fetchmail /var/run/fetchmail
 # Configures Postfix
 COPY target/postfix/main.cf target/postfix/master.cf /etc/postfix/
 COPY target/postfix/header_checks.pcre target/postfix/sender_header_filter.pcre target/postfix/sender_login_maps.pcre /etc/postfix/maps/
-RUN echo "" > /etc/aliases
+RUN touch /etc/aliases
```

Comment:
It was not clear to me why that line is needed at all.  I made the assumption that the goal was to create the file anew, since it's the Dockerfile here.  I changed it because I thought the old way was clunky.

I would have liked to have the defaults put in here but I don' think the environment for $HOSTNAME and $DOMAIN are exposed inside the Dockerfile?  I had tested versions that did things like:

amavis: postmaster

...etc...but that syntax in /etc/aliases forced the mail to get delivered locally (/var/mail) or fail if there is no user (i.e. postmaster).  Only if you specify something after @, will it deliver through SMTP.

Reply:
I understand. I'd propose the following to get the original functionality without the clunkyness:

``` DOCKERFILE
RUN : >/etc/aliases
```

Reply:
I'm not a bash user, really.  If you say it does that, then I like its cleanliness.

How do I update it now that it's already a PR?  Just push this change to the same branch?

Reply:
> I'm not a bash user, really.  If you say it does that, then I like its cleanliness.

I just wrote a little test Dockerfile - my suggested solution should work.

> How do I update it now that it's already a PR?  Just push this change to the same branch?

Exactly:)